saleor/graphql/page/sorters.py

```diff
@@ -19,6 +20,10 @@ def description(self):
             description = f"Sort pages by {sort_name}."
             if self.name == "PUBLICATION_DATE":
                 description += DEPRECATED_IN_3X_INPUT
+            if self.name == "CREATION_DATE":
+                description += DEPRECATED_IN_3X_INPUT
+            if self.name == "CREATED_AT":
+                description += ADDED_IN_38
```

Comment:
There is no index:
https://github.com/saleor/saleor/blob/c28b12323141208257338c3619a57bd2dc8b1922/saleor/page/models.py#L26